solutions/swb-app/src/accountRoutes.ts

```diff
@@ -15,7 +15,8 @@ import {
   UpdateAccountRequest,
   UpdateAccountRequestParser,
   GetAccountRequest,
-  GetAccountRequestParser
+  GetAccountRequestParser,
+  isInvalidAwsAccountIdError
```

Comment:
Nit: Pascal casing

Reply:
Its a function, not a class, so wouldn't it still be camel case?